DNA-RNA.py

```diff
@@ -0,0 +1,58 @@
+seq = str(input()).upper()
```

Comment:
What do have against lower case? Is it necessary, or just your preference?

Reply:
I use uppercase in my dictionary and I'm used to seeing nucleotides in uppercase